WyattHWilliams/react-challenge-project-jan-2022
# #5 Add-Feature: Logout Functionality

## Changes
1. fix authReducer so that login information is logged correctly.
2. add function in Nav that sends dispatch to the logoutUser function in authActions

## Purpose
previously the logout button did nothing except re-route the user to the login page.
addresses [issue 5](https://github.com/Shift3/react-challenge-project-jan-2022/issues/5)

## Approach
the logoutUser function in authActions was previously setup to clear the information in the reducer by setting it back to the initial state. By using that functionality, we clear the user's data. Without the token or email information, the user is effectively locked out of API calls to the server that require the token.

## Pre-Testing TODOs
1. login the user
2. check that the authReducer is storing the login information

## Testing Steps
1. click the logout button
2. check that the user is no on the logout page and that the authReducer has been set back to initial state.

Closes #005
